web/js/index.js

```diff
@@ -50,16 +52,22 @@ const initThreeJsElements = function() {
     controls.noZoom = false;
     controls.noPan = false;
 
-    const $viewport = document.getElementById('gltf-viewport');
     $viewport.appendChild(renderer.domElement);
-
-    window.addEventListener('resize', () => {
-        camera.aspect = window.innerWidth / window.innerHeight;
+    
+    /**
+     * Handles resizing the window.
+     */
+    const handleResize = () => {
+        const width = window.innerWidth,
+            height = (window.innerHeight - document.getElementById('elem-selector').offsetHeight);
+        camera.aspect = width / height;
         camera.updateProjectionMatrix();
-        renderer.setSize(window.innerWidth, window.innerHeight);
+        renderer.setSize(width, height * 0.9, false);
```

Comment:
Where does the 0.9 come from? Also I imagine it should only be defined once instead of hard coded. Same goes for 'elem-selector'.

Reply:
Updated.